src/sentry/interfaces/stacktrace.py

```diff
@@ -116,6 +122,11 @@ def remove_filename_outliers(filename):
     return _filename_version_re.sub('<version>/', filename)
 
 
+def remove_module_outliers(module):
+    """Remove things that augment the module but really should not."""
+    return _java_enhancer_re.sub('', module)
```

Comment:
I'd argue that we should probably replace with something like, `$$FooCGLIB$$<autogen>` rather than just completely yanking it out. This leaves the CGLIB name, and just replaces the hash with `<autogen>`